CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.3.0 (TBD)
 
+* [BREAKING] Simplified account details for on-chain accounts, now only delta provided
```

Comment:
minor nit: let's add a PR number to this line (similar to how we do it for other changes below).